### PR TITLE
Fix handling scoped packages in preset-env include/exclude options

### LINKED ISSUE #9182: @babel/preset-env exclude option doesn't support full package names
## Bug Report
The [`exclude` docs](https://babeljs.io/docs/en/babel-preset-env#exclude) themselves say
> This option is useful for "blacklisting" a transform like @babel/plugin-transform-regenerator

But `exclude: ["@babel/plugin-transform-regenerator"]` doesn't work; only `exclude: ['transform-regenerator']` seems to work.  If this is intentional the docs should clarify that this is a surprising difference from `presets` and `plugins` accepting full package names.

**Current Behavior**

```
{ Invariant Violation: [BABEL] /Users/andy/flow-runtime/packages/babel-plugin-flow-runtime/src/ConversionContext.js:
Invalid Option: The plugins/built-ins '@babel/plugin-transform-regenerator' passed to the 
'exclude' option are not valid. Please check data/[plugin-features|built-in-features].js in babel-preset-env
```

**Input Code**
(irrelevant)

**Expected behavior/code**

Babel CLI doesn't throw an error

**Babel Configuration (.babelrc, package.json, cli command)**

```js
{
  "presets": [
    ["@babel/preset-env", {
      "targets": {
        "browsers": ["last 2 versions"]
      },
      "exclude": [
        "@babel/plugin-transform-regenerator"
      ]
    }],
    "@babel/preset-react",
    "@babel/preset-flow"
  ],
  "plugins": [
    "@babel/plugin-proposal-class-properties",
    "@babel/plugin-proposal-object-rest-spread",
    ["@babel/plugin-proposal-decorators", {"legacy": true}]
  ]
}

```

**Environment**
- Babel version(s): 7.2.0
- Node/npm version: Node 8 (irrelevant)
- OS: OS X (irrelevant)
- Monorepo: yes, yarn repo
- How you are using Babel: cli

### PR DESCRIPTION
**Pull request by existentialism**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | Fixes #9182
| Patch: Bug Fix?          | Y
| Major: Breaking Change?  | N
| Minor: New Feature?      | N
| Tests Added + Pass?      | Yes
| Documentation PR Link    | 
| Any Dependency Changes?  |
| License                  | MIT

### CODE REVIEW

**Comment by nicolo-ribaudo** [comment]
We also need to support the `@babel/transform-classes` format

**Comment by existentialism** [comment]
@nicolo-ribaudo updated!

File: packages/babel-preset-env/src/normalize-options.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ const validBrowserslistTargets = [
 ];
 
 export const normalizePluginName = (plugin: string): string =>
-  plugin.replace(/^babel-plugin-/, "");
+  plugin.replace(/^@?babel[-/](plugin-)?/, "");
```
Comment (nicolo-ribaudo) [inline packages/babel-preset-env/src/normalize-options.js]: This also matches things like `@babel-plugin-foo`, is it intended to keep the RegExp simple?

Comment (existentialism) [reply packages/babel-preset-env/src/normalize-options.js]: Tweaked it to prevent `@babel-plugin-foo`

**Pull request merged**